### ISSUE TITLE
Incomplete ticket subject

### ISSUE THREAD

**Issue by nekvindao** [osTicket/osTicket-1.7#321]
If we receive an email, which has diacritics in its subject,  the ticket subject  is not displayed correctly.  

We only get the beginning of subject  and everything, which is written after the letter with diacritics is not  displayed.

Is this a known bug?
Thanks

**Comment by fabiorauber** [comment]
Also happening to me with accented characters. Everything written after the accented characters is not added to the title.

**Comment by fabiorauber** [comment]
Tried adding mime_encode to the ticket subject at the line 389 of class.mailfetcher.php. Now the title is complete, but the accents are not being displayed correctly. Anyone had a better experience with fixing this bug?

**Comment by greezybacon** [comment]
@fabiorauber, did you do a git pull before retrying and making that last comment?

**Comment by fabiorauber** [comment]
Yes, it was synchronized with the latest commits from osTicket repository. Fixed the problem by using iconv_mime_decode (don't know how to attach files in here):

a/include/class.mailfetch.php:

```
//Decode using iconv_mime_decode - handles accented characters better
 function mime_decode($text) {
    return iconv_mime_decode($text, 0, "UTF-8");
 }
```

**Comment by greezybacon** [comment]
Is this still a problem in the develop and master branches?
